src/modules/reminder-picker/reminder-picker.tsx

```diff
@@ -52,12 +56,20 @@ export const ReminderPicker: React.FC<IProps> = React.memo(
       onChange(Moment(date).format('YYYY-MM-DD'));
     };
 
+    const calendarPickerMinDate: Date =
+      // By default, allow selection starting tomorrow
+      minDate || Moment(new Date()).add(1, 'day').toDate();
+
     return (
       <StyledWrapper>
         <StyledSelectPickerWrapper>
+          <StyledLabel variant="caption" muted>
+            Set Reminder For{' '}
+            <Text color="dangerLight">{isRequired ? '*' : ''}</Text>
+          </StyledLabel>
+
           <SelectPicker
             {...props}
-            label="Set Reminder For"
```

Comment:
Instead of adding separate label above SelectPicker, can we modify the SelectPicker Component and change the label style as per the requirements? 
Actually in our Ministry app, we are using this component in multiple places but the label style does not match with Ministry's sketch design.

Reply:
The SelectPicker component was designed in reference to [this Sketch](https://www.sketch.com/s/370752e1-5760-434d-93aa-2e7659ab0086/a/Ly2vWD) (Field Labels Section).

Initially, the label was limited to just a bold label but looks like a new variant has been added. If you can, can you please update the SelectPicker component to accept a prop "subtitleLabel" which, when true, will render the label with small and muted text. See below:

<img width="489" alt="Screenshot 2022-07-26 at 5 06 08 PM" src="https://user-images.githubusercontent.com/50396163/180996933-50e40aec-7bf2-4de4-839d-e881de62eef0.png">

Reply:
Okay then let me add this subtitleLabel boolean prop in SelectPicker component. and may be you can remove that custom label above it as it will not require now once the SelectPicker component is updated.